common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1316,11 +1317,21 @@ def grading_context(self):
         except UndefinedContext:
             module = self
 
+        graded_location_block_types = set(
+            cat for (cat, xblock_class) in XBlock.load_classes() if (
+                getattr(xblock_class, 'has_score', True) or getattr(xblock_class, 'has_children', True)
+            )
+        )
```

Comment:
You mean at the module level? I guess I don't really understand how extensions are loaded under the covers -- will `load_classes()` return the right thing if I ran this code at the time of module import?